nf_core/modules/lint/module_tests.py

```diff
@@ -107,6 +107,22 @@ def module_tests(_, module: NFCoreComponent):
                                             snap_file,
                                         )
                                     )
+                                if "versions" in str(snap_content[test_name]):
```

Comment:
actually this test will fail for the correct `snapshot(process.out)` (which is the case for kallisto/quant, so we need some more logic here.